sensors_node:
  input_topic: /sensors
  sensors_topic: /ecs/sensors

# ECS CHANGE DETECTOR PARAMETERS
ecs_data_acquisition:
  sensors_topic: /ecs/sensors
  trigger_topic: /ecs/trigger
  gridmap_topic: /ecs/map
  sensors:
    - name: temperature_sensor
      method: diffratio
      parameters: { "sensitivity_threshold": 1.4, "min_length": 40 }
      republish: 0
    - name: humidity_sensor
      method: diffratio
      parameters: { "sensitivity_threshold": 1.4, "min_length": 40 }
      republish: 0    
    - name: distance_sensor
      method: varratio
      parameters:
        { "sensitivity_threshold": 1.4, "window_length": 40, "var_stable": 0.1 }
      republish: 0

ecs_ibec:
  trigger_topic: /ecs/trigger
  camera_topic: /camera/color/image_raw/compressed
  decision_topic: /ecs/decision
  network_name: "resnext"
  network_device: "gpu"
  number_of_classes: 2
  class_mapping:
    "0": indoor
    "1": outdoor

# ECS SWITCHER PARAMETERS
ecs_rba:
  decision_topic: /ecs/decision
  behaviors:
    - name: indoor
      default: 1
      commands:
        - set, decision, indoor
        - set, test_param, 1
    - name: outdoor
      commands:
        - set, decision, outdoor

# GRID MAP PARAMETERS
#grid_map_topic: /ecs/grid_map
#
#grid_map_visualizations:
#  - name: temperature
#    type: flat_point_cloud
#    params:
#      layer: temperature
#      height: 0.1
